src/main/java/net/william278/velocitab/tab/PlayerTabList.java

```diff
@@ -26,6 +26,8 @@ public class PlayerTabList {
     private final Velocitab plugin;
     private final ConcurrentLinkedQueue<TabPlayer> players;
     private final ConcurrentLinkedQueue<String> fallbackServers;
+    private int indexheader = 0;
+    private int indexFooter= 0;
```

Comment:
```suggestion
    private int footerIndex = 0;
```
style/naming/variable declaration spacing
will also need to update these names in the methods below :)